src/components/MoneyRequestConfirmationListFooter.tsx

```diff
@@ -526,7 +527,7 @@ function MoneyRequestConfirmationListFooter({
                             return;
                         }
 
-                        if (!isPolicyExpenseChat) {
+                        if (!isPolicyExpenseChat && (!isTrackExpense || !policyForMovingExpensesID)) {
```

Comment:
Why is this different from Category field? Can we use `shouldNavigateToUpgradePath` here?